src/ram/src/ram.cpp

```diff
@@ -480,27 +444,27 @@ void RamGen::generate(const int bytes_per_word,
     array<dbBTerm*, 8> D;     // array for b-term for external inputs
```

Comment:
I would use a more descriptive name, like `d_bterms`.
@maliberty what is the convention for capitalization in snake case? Should it be `d_bterms` or can it be `D_bterms`?